content/webapp/types/exhibition-guides.ts

```diff
@@ -43,7 +34,7 @@ export type ExhibitionGuideBasic = {
   id: string;
   image?: ImageType;
   promo?: ImagePromo;
-  relatedExhibition: Exhibit | undefined;
+  relatedExhibition: RelatedExhibition | undefined;
```

Comment:
Is there a reason we're mixing `key?: Value` and `key: Value | undefined`? Extremely rare to need to use the latter, prefer the former

Reply:
I don't think so.

AIUI `key: Value | undefined` forces you to declare a value, but I don't think we need that here. Changed.